spec/models/account_spec.rb

```diff
@@ -102,6 +125,58 @@
     end
   end
 
+  describe "#application_token" do
+    let(:client_credentials) { double }
+    let(:oauth2_client) { double }
+    let!(:account) { Account.create!(synced_id: 123) }
+
+    it "returns a client credential token setup without default params" do
```

Comment:
this definitely requires VCRs, this is testing the implementation, not behavior.